src/components/menu/styles.ts

```diff
@@ -7,7 +7,7 @@ export const ElMenu = styled.div`
 
 export const ElMenuPopover = styled.div`
   position: absolute;
-  z-index: 98;
+  z-index: 100;
 `
 
 export const ElMenuList = styled.div`
```

Comment:
actually currently `Menu.Item` should under `Menu.Group` at the moment, because `Menu.List` could have multiple group, as the design imply in the menu `mixed` variants

Reply:
The designs are not necessarily dev-focused and there's no detailed specs for menus, so we need to be careful drawing too many conclusions about how we then implement things. This kind of work typically requires a bit of research, discussion and collaboration with Design.

For example, my reading of the ARIA docs suggests items can be direct descendants of the `role='menu'` element, they do not need to be descendants of a group.

Reply:
I will adjust the code to use div, as per discussion below